src/HDF5.jl

```diff
@@ -901,7 +901,7 @@ function Base.setindex!(parent::Union{File,Group}, val, path::AbstractString; pv
 end
 
 # Check existence
-function Base.haskey(parent::Union{File,Group}, path::AbstractString, lapl::Properties=Properties())
```

Comment:
Right now we have
```julia
    function Properties(id, class::hid_t = H5P_DEFAULT)
        p = new(id, class)
        finalizer(close, p) #Essential, otherwise we get a memory leak, since closing file with CLOSE_STRONG is not doing it for us
        p
    end
 Properties() = Properties(H5P_DEFAULT)

const DEFAULT_PROPERTIES = Properties(H5P_DEFAULT, H5P_DEFAULT)

```

Should we change this to 
```julia
    function Properties(id::hid_t = H5P_DEFAULT, class::hid_t = H5P_DEFAULT)
        p = new(id, class)
        finalizer(close, p) #Essential, otherwise we get a memory leak, since closing file with CLOSE_STRONG is not doing it for us
        p
    end


```
I.e. just use the default constructor directly? I don't think it matters either way, just a design/consistency decision.